exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/DrillDistributionTrait.java

```diff
@@ -27,6 +27,16 @@
 
 public class DrillDistributionTrait implements RelTrait {
 
+  public enum DistributionType {
```

Comment:
This class contains also inner classes in the end. Moving the only enum makes code more inconsistent. Could you please either revert this change, so all inner types would be at the end or move them all to the beginning of the file.